public/js/layers/dashboard.js

```diff
@@ -28,10 +28,10 @@ export function createDashboardLayer(font, level) {
         font.print(playerTrait.name, context, 16, LINE1);
         font.print(playerTrait.score.toString().padStart(6, '0'), context, 16, LINE2);
 
-        font.print('@x' + playerTrait.coins.toString().padStart(2, '0'), context, 96, LINE2);
+        font.print('x' + playerTrait.coins.toString().padStart(2, '0'), context, 96, LINE2);
 
         font.print('WORLD', context, 152, LINE1);
-        font.print(level.name, context, 160, LINE2);
+        font.print(level.name.toUpperCase(), context, 160, LINE2);
```

Comment:
Can we put the responsibility to uppercase everything on the font perhaps? I am not sure it is a great idea or how it could be nice without complicating the font class.

Perhaps we can extend the font class and create an UppercaseOnlyFont class that does this.

Reply:
Made font class force uppercase. Replaced x with ×.